btr-web/btr-main-app/cypress/e2e/pages/add-individual/control-of-majority-of-directors.cy.ts

```diff
@@ -1,6 +1,6 @@
 describe('pages -> Add individual', () => {
   beforeEach(() => {
-    cy.visit('/')
+    cy.visitHomePageNoFakeData()
```

Comment:
NOTE: doing 'cy.visit('/')' without any intercepts setup causes the general error modal to pop up and mess things up. Generally the tests where this is relevant finish before the modal has time to pop up and block the view of things, but occasionally the modal pops up before the test has time to check things and it causes a test failure